scripts/examples/python/mini-xenrt.py

```diff
@@ -15,9 +15,9 @@
 
 class Operation:
     def __init__(self):
-        raise Exception("this is supposed to be abstract, dummy")
+        raise NotImplementedError("this is supposed to be abstract, dummy")
```

Comment:
This exception does not allow arguments, instead apply my precious suggested change

Reply:
Few sites had argument ,stuck with it . Removed the argument as suggested commit 1186ee869e001c565ea95bc174f3d8b60d250c01